packages/back-end/src/integrations/Presto.ts

```diff
@@ -102,4 +115,53 @@ export default class Presto extends SqlIntegration {
   ensureFloat(col: string): string {
     return `CAST(${col} AS DOUBLE)`;
   }
+  async getInformationSchema(): Promise<InformationSchema[]> {
+    const defaultCatalog = this.params.catalog;
+
+    if (!defaultCatalog)
+      throw new MissingDatasourceParamsError(
+        "To view the information schema for a Presto dataset, you must define a default catalog. Please add a default catalog by editing the datasource's connection settings."
+      );
+
+    const sql = `SELECT
+        table_name,
+        table_catalog,
+        table_schema,
+        count(column_name) as column_count
+      FROM
+        ${defaultCatalog}.information_schema.columns
+      GROUP BY (table_name, table_schema, table_catalog)`;
+
+    const results = await this.runQuery(sql);
+
+    if (!results.length) {
+      throw new Error(`No tables found.`);
+    }
+
+    return formatInformationSchema(results as RawInformationSchema[], "presto");
+  }
+
+  async getTableData(
+    databaseName: string,
+    tableSchema: string,
+    tableName: string
+  ): Promise<{ tableData: null | unknown[]; refreshMS: number }> {
+    const sql = `SELECT
+        data_type,
+        column_name
+      FROM
+        ${databaseName}.information_schema.columns
+      WHERE
```

Comment:
@jdorn As I was looking over this one more, I noticed this `WHERE` was previously an `AND`. I've updated this to be a `WHERE`. It wiped your review.